### ISSUE TITLE
Add proper padding between icons.

### ISSUE THREAD

**Issue by fadeouter** [wbolster/nothing-to-say#12]
I am a perfectionist and I noticed that Nothing To Say icon has smaller horizontal padding than another icons in gnome-shell. I experienced this in 3.22 and 3.24. Please, could you add proper padding?

[http://i.imgur.com/MKOZ14h.png](screenshot)

**Comment by wbolster** [comment]
yes, grave bug indeed. :)  though the screenshot seems a bit off i see what you mean.

if you could dig up example code from a well-behaved extension that would help a lot.
